sourcegraph/about
# [A11y]: Comply text colors w/ WCAG standards > 'text-muted'

Closes #5538: Fixes 'text-muted' wherever used on a grey background (this violates WCAG color contrast standards).
![image](https://user-images.githubusercontent.com/59381432/177423210-1b5fe00d-f838-4f30-aefc-ffcbe5d3986c.png)

### Notes
- The new [Figma DLS](https://www.figma.com/file/o1QRtdQI0ozKq0n7ATrKlx/Marketing-DLS?node-id=3212%3A21096) shows a slightly darker grey HEX for most of these items anyway. So I brought the related colors in & updated the relevant `text-muted` instances.
- I reached out to some of our designers (Steven & Tracey), and asked if the [Handbooks Brand Color](https://handbook.sourcegraph.com/departments/engineering/design/brand_guidelines/color/) guidelines can be in sync with the new Figma DLS above to eliminate any confusion on our end.
- ~We may need another ticket to deal with DLS color updates.~ Ticket here #5559


### Changelog
- Updates `text-muted` instances to be in line with DLS & comply with WCAG color contrast standards.
- Reduces unneeded BEM css in files touched.

### Test
1. Ensure prettier has standardized the proposed changes.
2. Run an [ARC Toolkit test](https://docs.sourcegraph.com/dev/background-information/web/accessibility/how-to-arc-toolkit) to ensure changed colors don't violate requirement.
3. Ensure pages with removed BEM classes have a comparable appearance now.
